package.json

```diff
@@ -158,6 +158,8 @@
     "chai-subset": "^1.3.0",
     "coveralls": "^2.11.12",
     "css-loader": "^0.24.0",
+    "electron": "^1.3.5",
+    "electron-builder": "^6.4.0",
```

Comment:
Couldn't find any . I can see `gulp-atom-electron` though. Are you referring to that ?